pyuvdata/uvdata/uvdata.py

```diff
@@ -862,17 +862,17 @@ def _look_in_catalog(
                 "sidereal" (fixed RA/Dec),
                 "ephem" (RA/Dec that moves with time),
                 "driftscan" (fixed az/el position),
-                "unphased" (no w-projection, equivalent to `phase_type` == "drift").
+                "unprojected" (no w-projection, equivalent to `phase_type` == "drift").
         cat_lon : float or ndarray
             Value of the longitudinal coordinate (e.g., RA, Az, l) of the phase center.
-            No default, not used when `cat_type="unphased"`. Expected to be a float for
-            sidereal and driftscan phase centers, and an ndarray of floats of shape
-            (Npts,) for ephem phase centers.
+            No default unless `cat_type="unprojected"`, in which case the default is
+            zero. Expected to be a float for sidereal and driftscan phase centers, and
+            an ndarray of floats of shape (Npts,) for ephem phase centers.
         cat_lat : float or ndarray
             Value of the latitudinal coordinate (e.g., Dec, El, b) of the phase center.
-            No default, not used when `cat_type="unphased"`. Expected to be a float for
-            sidereal and driftscan phase centers, and an ndarray of floats of shape
-            (Npts,) for ephem phase centers.
+            No default unless `cat_type="unprojected"`, in which case the default is
+            pi/2. Expected to be a float for sidereal and driftscan phase centers, and
+            an ndarray of floats of shape (Npts,) for ephem phase centers.
```

Comment:
Caught another glorious misspelling of mine below in the description for "cat_frame" ("unites" instead of "units"), if you'd be so kind...